src/inmanta/server/services/compilerservice.py

```diff
@@ -515,9 +515,14 @@ class CompilerService(ServerSlice, environmentservice.EnvironmentListener):
     def __init__(self) -> None:
         super().__init__(SLICE_COMPILER)
         self._recompiles: dict[uuid.UUID, Task[None | Result]] = {}
+
         self._global_lock = asyncio.locks.Lock()
+        # boolean indicating when everything is up and processing can start
```

Comment:
```suggestion
        # boolean indicating when everything is up and compile execution can start
```
If I understand correctly this determines only if we are ready to start executing compiles, not whether the service is ready to receive requests or anything like that. Right?

Reply:
correct